rosa_cluster_admin/rosa-cluster-autoscaling.adoc

```diff
@@ -0,0 +1,22 @@
+:_content-type: ASSEMBLY
+[id="rosa-cluster-autoscaling"]
+= Cluster autoscaling
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: rosa-cluster-autoscaling
+
+toc::[]
+
+Applying autoscaling to an {product-title} cluster involves deploying a cluster autoscaler and then deploying machine autoscalers for each machine type in your cluster.
```

Comment:
Specifically the word 'deploying' sounds incorrect because those objects are pretty much logical and not that much physical, maybe 'configuring' or 'setting'?